Only in containers-0.3.0.0: containers.patch
diff -ru containers-0.3.0.0/Data/Sequence.hs containers-0.3.0.0-new/Data/Sequence.hs
--- containers-0.3.0.0/Data/Sequence.hs	2009-11-19 07:51:36.000000000 -0800
+++ containers-0.3.0.0-new/Data/Sequence.hs	2010-07-21 13:42:30.000000000 -0700
@@ -1501,7 +1501,7 @@
 				foldrWithIndex g n xs)
 #else
 findIndicesL p xs = foldrWithIndex g [] xs where
-g i x is = if p x then i:is else is
+    g i x is = if p x then i:is else is
 #endif
 
 {-# INLINE findIndicesR #-}
@@ -1513,7 +1513,7 @@
 				foldlWithIndex g n xs)
 #else
 findIndicesR p xs = foldlWithIndex g [] xs where
-g is i x = if p x then i:is else is
+    g is i x = if p x then i:is else is
 #endif
 
 ------------------------------------------------------------------------
